packages/app/lib/home/pages/home_page.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:core/core.dart';
```

Comment:
@jenshor I'm not sure about the folder structure. Do you think it's okay this way?
To give you some context, my idea is that there will be a feature named `feed` in the `packages` folder which is responsible for getting a list of tweets. wen given a type (home, profile, search, ..etc)
And inside the `app` package and it's `home` folder, we would have 
* `home_page.dart`
* `home_feed_provider.dart` which will retrieve the home specific feed from the `feed` package.

Also for now I am passing the `user` and `homeFeed` from outside the `HomePage` widget just to be able to implement this in the fastest way with dummy data. Eventually, we will have providers and override them in `app_widgetbook`. Would love to discuss this further in a meeting.

Reply:
That works. In the end, I think it's best if you can just inject the data from outside of the Widget. That makes it completely independent of logic and extremely easy to test and to inject into Widgetbook.